### PR TITLE
Prepare next release entry for L2

### PR DESCRIPTION
**Pull request by ajsantander**

### CODE REVIEW

File: publish/releases.json [hunk 1]
```diff
@@ -236,5 +236,15 @@
...
 		"sips": [112, 136]
+	},
+	{
+		"name": "TBD",
+		"version": {
+			"major": 2,
+			"minor": 46
+		},
+		"ovm": true,
+		"sources": ["DebtCache"],
```
Comment (justinjmoses) [inline publish/releases.json:247]: It was actually more than this IIRC - it's everything that SIP-136 affected, which also includes an empty `EtherWrapper`. So I think you ned `DebtCache`, `Issuer` and `EtherWrapper` (which needs to be flipped to use `EmptyEtherCollatereal` (cause it just needs `totalIssuedSynths` to be `0` for https://github.com/Synthetixio/synthetix/blob/master/contracts/BaseDebtCache.sol#L215

File: publish/releases.json [hunk 2]
```diff
@@ -236,5 +236,15 @@
...
+		"sips": []
```
Comment (justinjmoses) [inline publish/releases.json:248]: please include `136` in here

File: publish/releases.json (hunk 1 above)
Comment (ajsantander) [reply publish/releases.json]: Done

File: publish/releases.json (hunk 2 above)
Comment (ajsantander) [reply publish/releases.json]: Done

**Review by justinjmoses — APPROVED** [review]
Let's just not merge this yet, as an L1 release may supersede it.

File: publish/releases.json [hunk 3]
```diff
@@ -236,5 +236,15 @@
...
 		"sips": [112, 136]
+	},
+	{
+		"name": "TBD",
+		"version": {
+			"major": 2,
+			"minor": 46
+		},
+		"ovm": true,
+		"sources": ["DebtCache", "Issuer", "EtherWrapper"],
```
Comment (justinjmoses) [inline publish/releases.json:247]: ah we also need a new `SystemSettings`- as it's a property that is checked in deployment

Comment (justinjmoses) [reply publish/releases.json:247]: see https://github.com/Synthetixio/synthetix/blob/master/publish/src/commands/deploy.js#L2269-L2292

**Comment by ajsantander** [comment]
deprecated

**Pull request closed without merging**